content/manual-references.json

```diff
@@ -942,4 +942,26 @@
     "type": "webpage",
     "standard_citation": "url:https://nihcc.app.box.com/v/ChestXray-NIHCC"
   }
+{
```

Comment:
I believe the Travis CI build failed because the previous line (944) needs to be terminated with a `,`.  Can you please see if that fixes the JSON error?

Reply:
Indeed. Would help to use a text editor that lints JSON automatically to catch issues like this.

Also please use same indentation as the citation above.